docs/recipes/cache.md

```diff
@@ -110,7 +110,10 @@ export class NovelComponent implements OnDestroy {
         switchMap(params =>
           store
             .dispatch(new GetNovelById(params.id))
-            .pipe(mapTo(store.selectSnapshot(NovelsInfoState.getNovelById(params.id))))
+            .pipe(
+              mapTo(store.select(NovelsInfoState.getNovelById(params.id))),
+              mergeMap(novel => novel)
+            )
```

Comment:
Better:
```suggestion
            .pipe(
              mergeMap(() => store.select(NovelsInfoState.getNovelById(params.id))),
            )
```

Reply:
Hmmm. github does not support multiline suggestions but hopefully the suggestion is obvious